spec/lcm/integration/shared_examples_for_synchronization_bricks.rb

```diff
@@ -132,9 +127,10 @@
   end
 
   it 'migrates metrics' do
-    used_in_dashboard = 'metric.customers.count'
-    tagged_production = Support::PRODUCTION_TAGGED_METRIC
-    expected_metrics = [used_in_dashboard, tagged_production]
+    used_reports = original_project.dashboards.map{|d|d.tabs.map{|t|t.items.map{|i|i}}}.flatten.map{|r|r.obj}.uniq{|r|r.identifier}
+    used_metrics = used_reports.map(&:definition).map(&:metrics).flatten.uniq{|m|m.identifier}
+    tagged_production = original_project.metrics.select { |m| m.tags.include?("metric") }
+    expected_metrics = used_metrics.concat(tagged_production).uniq{|m|m.identifier}.map(&:identifier)
```

Comment:
Style/SymbolProc: Pass `&:identifier` as an argument to `uniq` instead of a block.